### PR TITLE
One sentence per line rule for items list

### PR DESCRIPTION
**Pull request by fabienrohrer**
This is the last big coding style refactor I would like to apply on the doc.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by DavidMansolino** [comment]
👀

File: reference/hinge2joint.md [hunk 1]
```diff
@@ -31,15 +31,10 @@ In other words, this joint cannot be statically based.
...
-[HingeJointParameters](hingejointparameters.md) node It contains, among others,
-the joint position, the axis position expressed in relative coordinates, the
-stop positions and suspension parameters. If the `jointParameters` field is left
-empty, default values of the HingeJointParameters node apply.
-
-- `device2`: This field optionally specifies a
-[RotationalMotor](rotationalmotor.md), an angular
-[PositionSensor](positionsensor.md) and/or a [Brake](brake.md) device attached
-to the second axis. If no motor is specified, this part of the joint is passive.
+- `jointParameters2`: This field optionally specifies a [HingeJointParameters](hingejointparameters.md) node It contains, among others, the joint position, the axis position expressed in relative coordinates, the stop positions and suspension parameters.
```
Comment (DavidMansolino) [inline reference/hinge2joint.md]: Not reelated to this PR but a point is missing here: `...node It contai...`

File: reference/gyro.md [hunk 2]
```diff
@@ -19,24 +19,17 @@ The angular velocity is measured in radians per second [rad/s].
...
-should be enabled or disabled for the specified axis. If one of these fields is
-set to FALSE, then the corresponding vector element will not be computed and it
-will return *NaN* (Not a Number) For example if `zAxis` is FALSE, then
-the second element of the array returned by the `wb_gyro_get_values` function returns *NaN*.
+- `lookupTable`: This field optionally specifies a lookup table that can be used for mapping the raw angular velocity values [rad/s] to device specific output values.
+With the lookup table it is also possible to add noise and to define the min and max output values.
+By default the lookup table is empty and therefore the raw values are returned (no mapping).
+
+- `xAxis, yAxis, zAxis`: Each of these boolean fields specifies if the computation should be enabled or disabled for the specified axis.
+If one of these fields is set to FALSE, then the corresponding vector element will not be computed and it will return *NaN* (Not a Number) For example if `zAxis` is FALSE, then the second element of the array returned by the `wb_gyro_get_values` function returns *NaN*.
```
Comment (DavidMansolino) [inline reference/gyro.md]: Not reelated to this PR but a point is missing here: `...umber) For exampl...`

File: reference/camera.md [hunk 3]
```diff
@@ -31,109 +31,85 @@ Depending on its setup, the Camera node can model a linear camera, a typical RGB
...
-view can be computed from the `width`, `height` and horizontal `fieldOfView`:
+- `fieldOfView`: horizontal field of view angle of the camera.
+The value is limited to the range 0 to π radians if the `spherical` field is set to FALSE, otherwise there is no upper limit.
+Since camera pixels are squares, the vertical field of view can be computed from the `width`, `height` and horizontal `fieldOfView`:
 
     *vertical FOV = fieldOfView * height / width*
 
 - `width`: width of the image in pixels
 
 - `height`: height of the image in pixels
```
Comment (DavidMansolino) [inline reference/camera.md]: Here the final points are missing

**Comment by fabienrohrer** [comment]
Thank you for the review. I fixed these issues.

👍

**Pull request merged**